### ISSUE TITLE
quilc in the official Forest SDK RPMs and Linux tarball does not run

### ISSUE THREAD

**Issue by refi64** [rigetti/quilc#461]
```
$ quilc -S
Unhandled CFFI:LOAD-FOREIGN-LIBRARY-ERROR in thread #<SB-THREAD:THREAD "main thread" RUNNING
                                                       {10006704F3}>:
  Unable to load any of the alternatives:
   (#P"/src/quilc/src/contrib/tweedledum/libtweedledum.so" "libtweedledum.so"
    "tweedledum.so")

Backtrace for: #<SB-THREAD:THREAD "main thread" RUNNING {10006704F3}>
0: (SB-DEBUG::DEBUGGER-DISABLED-HOOK #<CFFI:LOAD-FOREIGN-LIBRARY-ERROR "Unable to load any of the alternatives:~%   ~S" {1004756623}> #<unused argument> :QUIT T)
1: (SB-DEBUG::RUN-HOOK *INVOKE-DEBUGGER-HOOK* #<CFFI:LOAD-FOREIGN-LIBRARY-ERROR "Unable to load any of the alternatives:~%   ~S" {1004756623}>)
2: (INVOKE-DEBUGGER #<CFFI:LOAD-FOREIGN-LIBRARY-ERROR "Unable to load any of the alternatives:~%   ~S" {1004756623}>)
3: (ERROR CFFI:LOAD-FOREIGN-LIBRARY-ERROR :FORMAT-CONTROL "Unable to load any of the alternatives:~%   ~S" :FORMAT-ARGUMENTS ((#P"/src/quilc/src/contrib/tweedledum/libtweedledum.so" "libtweedledum.so" "tweedledum.so")))
4: (CFFI::FL-ERROR "Unable to load any of the alternatives:~%   ~S" (#P"/src/quilc/src/contrib/tweedledum/libtweedledum.so" "libtweedledum.so" "tweedledum.so"))
5: (CFFI::TRY-FOREIGN-LIBRARY-ALTERNATIVES CL-QUIL.TWEEDLEDUM::LIBTWEEDLEDUM (#P"/src/quilc/src/contrib/tweedledum/libtweedledum.so" "libtweedledum.so" "tweedledum.so") (#P"./" #P"/usr/local/lib/rigetti/" #P"/usr/local/lib/"))
6: ((FLET CFFI::%DO-LOAD :IN CFFI::%DO-LOAD-FOREIGN-LIBRARY) #<CFFI:FOREIGN-LIBRARY LIBTWEEDLEDUM> CL-QUIL.TWEEDLEDUM::LIBTWEEDLEDUM (:OR #P"/src/quilc/src/contrib/tweedledum/libtweedledum.so" "libtweedledum.so" "tweedledum.so"))
7: (CFFI:LOAD-FOREIGN-LIBRARY CL-QUIL.TWEEDLEDUM::LIBTWEEDLEDUM :SEARCH-PATH NIL)
8: (CL-QUIL.TWEEDLEDUM:LOAD-TWEEDLEDUM)
9: (QUILC::ENTRY-POINT ("quilc" "-S"))
10: ((LAMBDA NIL :IN MAKE-TOPLEVEL-FUNCTION))
11: ((FLET SB-UNIX::BODY :IN SAVE-LISP-AND-DIE))
12: ((FLET "WITHOUT-INTERRUPTS-BODY-14" :IN SAVE-LISP-AND-DIE))
13: ((LABELS SB-IMPL::RESTART-LISP :IN SAVE-LISP-AND-DIE))

unhandled condition in --disable-debugger mode, quitting
```

I noticed libtweedledum.so isn't present in the rpm:

```
$ rpm -ql forest-sdk
/usr/local/bin/quilc
/usr/local/bin/qvm
/usr/local/lib/rigetti/libzmq.so
/usr/local/lib/rigetti/libzmq.so.5
/usr/local/share/man/man1/quilc.1
/usr/local/share/man/man1/qvm.1
```

However, it *is* present in the barebones tarball. Despite this, an identical error appears anyway; setting `export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$HOME/rigetti/forest-sdk_2.12.1-linux-barebones` "fixed" it.

**Comment by appleby** [comment]
Thanks for reporting this @refi64. Just a heads-up: I've moved this issue from the qvm repository -> quilc, since this is a quilc packaging bug.

**Comment by appleby** [comment]
It looks like libtweedledum.so is missing from the rpm spec file, and hence is not included in the rpm build.

**Comment by refi64** [comment]
Was running it in a Fedora 30 container.

**Comment by notmgsk** [comment]
Fixed #475. Find the new packages at https://qcs.rigetti.com/sdk-downloads.
